### PR TITLE
New favicon

### LINKED ISSUE #6: Update favicon

### PR DESCRIPTION
**Pull request by crnicholson**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -29,6 +29,7 @@ yarn-error.log*
 .env.development.local
 .env.test.local
 .env.production.local
+.github
```
Comment (jaspermayone) [inline .gitignore]: @crnicholson this should not be in the gitignore.

File: README.md [hunk 2]
```diff
@@ -1,14 +1,30 @@
...
+# Ham Club - A New YSWS from Hack Club
+Ham Club is a program for teens in the US running from April 29th to May 31st, allowing youth to get their ham radio license and then receive a free ham radio!
+
+# You Ship, We Ship
+
+At Hack Club we're re-imagining a new type of public education for technology. We believe people learn best by creating things they care about and sharing them with others. We build open source creative tools with young people all around the world. These tools are gateways to new subjects in technology like [embedded systems](https://sprig.hackclub.com), [circuits](https://onboard.hackclub.com) or digital fabrication robotics. When teenagers build things with our tools we send them more creative materials for free. 
+
+Our goal is to build a new type of library online that offers learners not just media to consume but tools to create. And it is free and open source to all. We're just getting started...
+
+# Development
```
Comment (jaspermayone) [inline README.md]: do we need a development section?

File: README.md [hunk 3]
```diff
@@ -1,14 +1,30 @@
...
+
+Join the `#hamradio` channel on the [Hack Club Slack](https://hackclub.com/slack). 
+
+The Blot editor is run with an Express server.
```
Comment (jaspermayone) [inline README.md]: We aren't blot

File: components/head.js [hunk 4]
```diff
@@ -1,22 +1,24 @@
...
+  const description = 'Earn Your Callsign, Get a Free Radio.';
+  const url = 'https://ham.hackclub.com';
+  const iconUrl = '/ham2.jpg'; // Local favicon in public/
+
+  return (
+    <Head>
+      <meta charSet="utf-8" />
+      <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
+      <meta name="viewport" content="width=device-width, initial-scale=1" />
+      <title>{title}</title>
```
Comment (jaspermayone) [inline components/head.js:14]: there's super weird formatting here!

File: public/ham1.jpg
Comment (jaspermayone) [inline public/ham1.jpg:1]: Why do we need these hams?

File: public/ham2.jpg
Comment (jaspermayone) [inline public/ham2.jpg:1]: Why do we need these hams?

File: yarn.lock
Comment (jaspermayone) [inline yarn.lock:1]: We aren't using yarn, please don't check this in.

File: README.md [hunk 5]
```diff
@@ -1,14 +1,30 @@
...
+
+```
+git clone https://github.com/hackclub/ham-club/
+yarn install
+```
+
+To run the development server:
+
+```
+yarn dev
```
Comment (jaspermayone) [inline README.md]: we use npm not yarn

File: README.md [hunk 6]
```diff
@@ -1,14 +1,30 @@
...
+
+Join the `#hamradio` channel on the [Hack Club Slack](https://hackclub.com/slack). 
+
+The Blot editor is run with an Express server.
+
+After installing `Git`, `Node.js`, and `Yarn`, clone the repo.
+
+```
+git clone https://github.com/hackclub/ham-club/
+yarn install
```
Comment (jaspermayone) [inline README.md]: we use npm not yarn

File: README.md [hunk 7]
```diff
@@ -1,14 +1,30 @@
...
+
+Our goal is to build a new type of library online that offers learners not just media to consume but tools to create. And it is free and open source to all. We're just getting started...
+
+# Development
+
+Join the `#hamradio` channel on the [Hack Club Slack](https://hackclub.com/slack). 
+
+The Blot editor is run with an Express server.
+
+After installing `Git`, `Node.js`, and `Yarn`, clone the repo.
```
Comment (jaspermayone) [inline README.md]: we use npm not yarn

**Review by jaspermayone — CHANGES_REQUESTED** [review]

File: components/head.js (hunk 4 above)
Comment (crnicholson) [reply components/head.js:14]: It was similar earlier!

**Comment by jaspermayone** [comment]
@crnicholson will re review shortly

**Review by jaspermayone — CHANGES_REQUESTED** [review]
@crnicholson please go through and make sure you addressed all the comments I made in my last review, then re request it once you have.

**Pull request closed without merging**